app/controllers/NotesController.py

```
@@ -2,12 +2,19 @@
 
 class NotesController:
     def create(self, **kwargs) -> dict:
+        try:
+            if not kwargs.get("title") or not kwargs.get("text"):
+                return {"error": "title or text missing"}
```

raise Exception instead of returning error dict